pkg/kf/commands/buildpacks/buildpacks.go

```diff
@@ -32,9 +32,10 @@ func NewBuildpacks(p *config.KfParams, l buildpacks.BuildpackLister) *cobra.Comm
 		Args:  cobra.ExactArgs(0),
 		Long:  ``,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			bps, err := l.List(
-				buildpacks.WithBuildpackListNamespace(p.Namespace),
-			)
+			if p.Namespace != "default" && p.Namespace != "" {
```

Comment:
I we should change this negative into a positive. Something like:

"NOTE: Buildpacks are global and are available to all spaces."

That way when we support the `target` command to default your namespace this won't pop up unnecessarily.

Reply:
Good idea.